browser/main/NoteList/index.js

```diff
@@ -153,23 +195,31 @@ class NoteList extends React.Component {
     }
     let { router } = this.context
     let { location } = this.props
+    let { selectedNoteKeys, shiftKeyDown } = this.state
 
     let targetIndex = this.getTargetIndex()
 
-    if (targetIndex === this.notes.length - 1) {
+    if (targetIndex === this.notes.length - 1 && shiftKeyDown) {
+      return
+    } else if (targetIndex === this.notes.length - 1) {
```

Comment:
Please add some variables to keep value of `targetIndex === this.notes.length - 1`